<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChromiumWebSerf</class>
 <widget class="QDialog" name="ChromiumWebSerf">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>360</width>
    <height>450</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="pushDelCurProfile">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>370</y>
     <width>141</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Удалить настройку</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushSaveSetting">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>411</y>
     <width>137</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Сохранить настройку</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushCancellation">
   <property name="geometry">
    <rect>
     <x>263</x>
     <y>411</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Отмена</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushStart">
   <property name="geometry">
    <rect>
     <x>154</x>
     <y>411</y>
     <width>103</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Начать парсинг</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>9</y>
     <width>341</width>
     <height>111</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QComboBox" name="comboBoxLinks">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="locale">
       <locale language="English" country="UnitedStates"/>
      </property>
      <property name="editable">
       <bool>false</bool>
      </property>
      <property name="maxVisibleItems">
       <number>10</number>
      </property>
      <property name="frame">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="labelInfo">
      <property name="text">
       <string> Парсинг Java скриптов</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="comboBoxJSRegExp">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="focusPolicy">
       <enum>Qt::WheelFocus</enum>
      </property>
      <property name="toolTip">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Регулярные выражения для скриптов необходимо устанавливать в том порядке, в котором скрипты буду запускаться&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="editable">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushPrSettings">
      <property name="text">
       <string>Настроить парсинг JS</string>
      </property>
      <property name="checkable">
       <bool>false</bool>
      </property>
      <property name="autoDefault">
       <bool>false</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>12</x>
     <y>182</y>
     <width>341</width>
     <height>180</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QLabel" name="labelAddressSite">
      <property name="text">
       <string>Адрес сайта</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="urlEdit">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="labelSetNameSite">
      <property name="text">
       <string>Название профиля настроек</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEditFileName">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="labelSetDir">
      <property name="text">
       <string>Дирректория для сохранения собранных данных</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineDirToDBEdit">
      <property name="text">
       <string>D:/Download/saveSites/</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushSelectDir">
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
      <property name="text">
       <string>Выбрать дирректорию</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
      <property name="autoDefault">
       <bool>false</bool>
      </property>
      <property name="default">
       <bool>false</bool>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>comboBoxJSRegExp</sender>
   <signal>activated(QString)</signal>
   <receiver>ChromiumWebSerf</receiver>
   <slot>addRegExpForParseJS(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>131</x>
     <y>76</y>
    </hint>
    <hint type="destinationlabel">
     <x>5</x>
     <y>122</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushCancellation</sender>
   <signal>clicked()</signal>
   <receiver>ChromiumWebSerf</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>326</x>
     <y>421</y>
    </hint>
    <hint type="destinationlabel">
     <x>353</x>
     <y>375</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>addRegExpForParseJS(QString)</slot>
 </slots>
</ui>
